modules/persistent-storage-csi-sc-disable.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-disable_{context}"]
+= Disabling the default storage class
+
+Disabling the default storage class allows you to accomplish several different objectives:
+
+* Enforce static provisioning by disabling dynamic provisioning.
+
+* When you have other preferred storage classes, prevent the storage operator from re-creating the initial default storage class.
+
+* Rename the default storage class by deleting the original default storage class, and then creating a new default storage class with a custom name.
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+.Procedure
+To disable the default storage class
+
+. Log in to the web console.
+
+. Click *Administration* > *CustomResourceDefinitions*.
+
+. On the *CustomResourceDefinitions* page, type "clustercsidriver" to find the `ClusterCSIDriver` object.
```

Comment:
s/"clustercsidriver"/`clustercsidriver`. Typing "clustercsidriver" with the quotes doesn't work. Might cause someone a problem.